eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/framework/list/flexibleadapter/FlexibleAdapterNotifier.java

```diff
@@ -42,14 +42,17 @@ public FlexibleAdapterNotifier(FlexibleAdapter flexibleAdapter)
     @Override
     public void notifyInitialItemsAdded(List initialItems)
     {
-        //noinspection unchecked
-        mFlexibleAdapter.addItems(0, initialItems);
+        notifyNewItemsAdded(initialItems, 0);
     }
 
 
     @Override
     public void notifyNewItemsAdded(List newItems, int positionStart)
     {
+        if (newItems == null || newItems.isEmpty())
```

Comment:
Could newItems possibly be null here? I.e. do you ever willingly pass `null` to this method directly or indirectly?
If not, I'd rather see this fail here, because there is clearly something wrong.